anta/custom_types.py

```diff
@@ -204,3 +204,36 @@ def validate_regex(value: str) -> str:
 ]
 BgpUpdateError = Literal["inUpdErrWithdraw", "inUpdErrIgnore", "inUpdErrDisableAfiSafi", "disabledAfiSafi", "lastUpdErrTime"]
 BfdProtocol = Literal["bgp", "isis", "lag", "ospf", "ospfv3", "pim", "route-input", "static-bfd", "static-route", "vrrp", "vxlan"]
+
+
+class APISSLCertificate(BaseModel):
```

Comment:
If we start to "extract" the models from the tests themselves (which I agree to make the tests cleaner and lighter), maybe we should put them in a separate module?

Reply:
In this case, it is used as type in a test. It is just another custom type. I could also have let it in its test and write its unit test in the related unit test for the AntaTest, but this type is quite generic. It is an API SSL Profile for EOS.